src/keeper.ts

```diff
@@ -264,6 +292,88 @@ class Keeper {
         component: "Indexer",
       });
     });
+
+    // update volume metrics
+    await this.updateVolumeMetrics(deps);
```

Comment:
updateVolumeMetrics doesn't do any thing async. So can remove this `await`. (and make it a non async fn)

Reply:
:+1: